### PR TITLE
BDOG-650 add support for play 2.7

### PR DESCRIPTION
**Pull request by bennetimo**
1. Adds support for Play 2.7
1. Updates to sbt `1.3.9`
1. Bumps plugins and dependencies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: scala
```
Comment (stevenmallen) [inline .travis.yml:1]: Is this file still needed?  If so, please could you explain the reason for these alterations

File: build.sbt [hunk 2]
```diff
@@ -31,13 +31,14 @@ lazy val library = Project(libName, file("."))
   .settings(
     name := libName,
     scalaVersion        := "2.11.12",
-    crossScalaVersions  := Seq("2.11.12", "2.12.8"),
+    crossScalaVersions  := Seq("2.11.12", "2.12.10"),
```
Comment (stevenmallen) [inline build.sbt:34]: Does this mean we are removing support for Scala < 2.12.10?  Is this because they are not compatible with Play 2.7?

File: project/BuildDependencies.scala [hunk 3]
```diff
@@ -5,8 +5,9 @@ private object BuildDependencies {
 
   val compile: Seq[ModuleID] = dependencies(
     shared = Seq("com.iheart" %% "ficus" % "1.4.7"),
-    play25 = Seq("uk.gov.hmrc" %% "http-verbs" % "9.8.0-play-25" % Provided),
-    play26 = Seq("uk.gov.hmrc" %% "http-verbs" % "9.8.0-play-26" % Provided)
+    play25 = Seq("uk.gov.hmrc" %% "http-verbs-play-25" % "10.11.0" % Provided),
```
Comment (stevenmallen) [inline project/BuildDependencies.scala:8]: Is there any impact on users of the library currently using play 2.5 and play 2.6?  We're OK with dependency updates, but just want to understand the impact/reason.

File: .travis.yml (hunk 1 above)
Comment (bennetimo) [reply .travis.yml:1]: It's not needed if you don't care for Travis. We have it setup for many of our public repos for the early validation of PRs etc: https://travis-ci.org/github/hmrc/auth-client

File: build.sbt (hunk 2 above)
Comment (bennetimo) [reply build.sbt:34]: No as when picking out a cross version only the major/minor version is used, i.e. `2.12`. It just updates the version used for the cross-compilation process, which we generally do if it is a patch version

File: .travis.yml (hunk 1 above)
Comment (stevenmallen) [reply .travis.yml:1]: Thanks, Tim - we can keep it.

File: build.sbt (hunk 2 above)
Comment (stevenmallen) [reply build.sbt:34]: OK
